components/style/properties/properties.mako.rs

```diff
@@ -1141,9 +1141,8 @@ impl PropertyDeclaration {
                   matches!(*value, DeclaredValue::WithVariables(_))
               },
           % endfor
-          PropertyDeclaration::Custom(_, ref value) => match *value {
-              DeclaredValue::CSSWideKeyword(_) => false,
-              _ => true,
+          PropertyDeclaration::Custom(_, ref value) => {
+            matches!(*value, DeclaredValue::CSSWideKeyword(..))
```

Comment:
note that this should be `!matches`, not just `matches`, you want to return `true` if it matches anything _but_ `CSSWideKeyword`, this is doing the opposite.